src/renderers/shared/shared/ReactTreeTraversal.js

```diff
@@ -90,13 +91,54 @@ function getParentInstance(inst) {
 /**
  * Simulates the traversal of a two-phase, capture/bubble event dispatch.
  */
+
+function isInteractive(inst) {
+  var tag = ReactDOMFeatureFlags.useFiber ? inst.type : inst._tag;
+
+  return (
+    tag === 'button' || tag === 'input' ||
+    tag === 'select' || tag === 'textarea' ||
+    tag === 'fieldset'
+  );
+}
+
+function shouldIgnoreElement(inst) {
+  if (inst && isInteractive(inst)) {
+    if (ReactDOMFeatureFlags.useFiber) {
```

Comment:
Hmm. `ReactTreeTraversal` uses this check: 

https://github.com/facebook/react/blob/master/src/renderers/shared/shared/ReactTreeTraversal.js#L20

```
  if (typeof inst.tag === 'number') {
```

Is that the preferred route? Would it be worth making that a more established convention?